README.md

```diff
@@ -1584,6 +1584,7 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 - [LimeSurvey](https://www.limesurvey.org) - Feature-rich Open Source web based polling software. Supports extensive survey logic. ([Demo](https://demo.limesurvey.org), [Source Code](https://github.com/LimeSurvey/LimeSurvey)) `GPL-2.0` `PHP`
 - [Meetable](https://meetable.org) - Event aggregator. ([Demo](https://events.indieweb.org), [Source Code](https://github.com/aaronpk/Meetable)) `MIT` `PHP`
 - [Mobilizon](https://mobilizon.org) - A federated tool that helps you find, create and organise events and groups. ([Demo](https://demo.mobilizon.org/), [Source Code](https://framagit.org/framasoft/mobilizon/)) `GPL-3.0` `Elixir`
+- [OhMyForms](https://ohmyform.com/) - Open source alternative to TypeForm, TellForm, or Google Forms. ([Source Code](https://github.com/ohmyform/ohmyform)) `AGPL-3.0` `Docker`
```

Comment:
The description should explain what the software does. You could replace it with the description from their README:

> An open source alternative to TypeForm that can create stunning mobile-ready forms, surveys and questionnaires.